libs/kano-profile/includes/kano_profile_bindings.h

```diff
@@ -29,12 +29,25 @@ namespace kano_profile {
     {
         public:
             apps();
+            void finalise();
```

Comment:
If you change this class to be `class apps : public Binding` and change the `void finalise()` in the `Binding` base class then you can do away with requiring a re-implementation of `finalise()` which just calls the protected method.